dashboards/google-compute-engine/vm-lifecycle-dashboard.json

```diff
@@ -105,7 +105,27 @@
           },
           "width": 9,
           "yPos": 3
-        }
-      ]
+      },
+      {
+        "height": 2,
+        "widget": {
+          "text": {
+            "content": "To use this dashboard a user-defined log-based metric using this project's Cloud Audit Logs is required. For more more information on how to use this dashboard  see [Monitor VM lifecycle events](https://cloud.devsite.corp.google.com/compute/docs/troubleshooting/troubleshooting-reboots#monitor-events).",
```

Comment:
Some nits:
1. "To use this dashboard, a user-defined log-based metric" (add comma)
2. "For more information" (delete double "more")
3. "on how to use this dashboard, see" (replace double space with comma space)
4. Should this be the non devsite version of the URL? The devsite URLs are not publicly available, right?

Reply:
Sorry for the super long delay. I've applied your proposed changes.